docs/addons.md

```diff
@@ -0,0 +1,2 @@
+\[../addons/README.rst\](../addons/README.rst)
```

Comment:
Looks like these links got escaped, but that shouldn't be a problem since we're redoing our role READMEs.